scriptworker/task.py

```diff
@@ -329,3 +304,24 @@ def max_timeout(context, proc, timeout):
         asyncio.ensure_future(kill(-pid)),
         asyncio.ensure_future(kill(pid))
     ]))
+
+
+async def download_artifacts(context, file_urls, download_func=download_file):
```

Comment:
I was thinking about putting this and download_file in scriptworker.client since those are more shared with client scripts, and that module could become a standalone module at some point.  I'm not sure if that's a valuable convention, yet, so we don't have to go that way.